digital_image_processing/filters/local_binary_pattern.py

```diff
@@ -0,0 +1,96 @@
+import cv2
+import numpy as np
+
+
+def get_neighbors_pixel(
+    image: np.ndarray, x_coordinate: int, y_coordinate: int, center: int
+) -> int:
+    """
+    Comparing local neighborhood pixel value with threshold value of centre pixel.
+    Exception is required when neighborhood value of a center pixel value is null.
+    i.e. values present at boundaries.
+
+    :param image: The image we're working with
+    :param x_coordinate: x-coordinate of the  pixel
+    :param y_coordinate: The y coordinate of the pixel
+    :param center: center pixel value
+    :return: The value of the pixel is being returned.
+    """
+    value = 0
+
+    try:
+        if image[x_coordinate][y_coordinate] >= center:
+            value = 1
+    except Exception:
+        pass
+
+    return value
```

Comment:
Sorry.  I misread the comments.  Please:
1. if center pixel value is null then return 0
2. `return int(image[x_coordinate][y_coordinate] >= center)`

Reply:
```
if center is None:
     return 0

return int(image[x_coordinate][y_coordinate] >= center)
```

IndexError: index 512 is out of bounds for axis 0 with size 512

Reply:
try:
    return int(image[x_coordinate][y_coordinate] >= center)
except (IndexError, TypeError):
    return 0